apps/voting/app/src/components/VotingRow.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Button, Countdown, TableCell, TableRow } from '@aragon/ui'
+import { VOTE_UNKNOWN } from '../vote-types'
+import ProgressBar from './ProgressBar'
+import VotingStatus from './VotingStatus'
+
+class VotingRow extends React.Component {
```

Comment:
We can do this later, but it'd be interesting to have a interval like `Countdown` that changes the style of a vote that was previously open but then expired while the app was open.

There's a lot of states for this though, e.g. where a vote might become approved while the app is open but we'd only learn of it via a state propagation from the wrapper.

Reply:
Yes I agree, it would be nice to make the UI react to any change in the state.